### PR TITLE
Allow to sort the table columns

### LINKED ISSUE #166: [Drain Safe] Allow to order list elements by name, value or USD value
## Overview
Currently we show the whole token list ordered as we get it from the backend. It can be really handy for the user to be able to sort tokens by name or value, as once we allow the user to select which tokens to drain the user could be interested to export only the most valuable tokens

## Goals
Enable ordering the table using the header options, showing the user the values ordered by each of the table columns

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves
Resolves #166

## How this PR fixes it
We can now order the table ascending and descending for any column clicking on the header titles or icons

## How to test it
Add the custom app using the deployed site generated in this PR to a Gnosis Safe Web (any environment)

## Screenshots

https://user-images.githubusercontent.com/1576776/140061530-0d13ea4c-807e-425e-a5dd-4e023c25eeac.mov

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1987,6 +1987,12 @@
     classnames "^2.2.6"
     react-media "^1.10.0"
 
+"@gnosis.pm/safe-react-components@git+https://github.com/gnosis/safe-react-components#1d732d38f1f7a6497b404339f6b75f9aeda6da39":
+  version "0.8.0"
+  resolved "git+https://github.com/gnosis/safe-react-components#1d732d38f1f7a6497b404339f6b75f9aeda6da39"
```
Comment (dasanra) [inline yarn.lock:1992]: There are some inconsistencies in the yarn.lock this is still pointing to a github commit.

Maybe running `yarn install` is still needed to grant the sync

Comment (yagopv) [reply yarn.lock]: yep 🙃

Comment (yagopv) [reply yarn.lock]: fixed

**Review by dasanra — APPROVED** [review]

**Comment by JagoFigueroa** [comment]
All good, thank you!

Yago also mentioned that the arrow will be removed from the 'exclude' list when a pending pr to the component library is merged as well :)

**Pull request merged**